http/src/main/resources/reference.conf

```diff
@@ -258,6 +259,22 @@ azure {
          path = "/"
        }
      ]
+     enabled = true
+   }
+
+   hail-batch-app-config {
+     chart-name = "/leonardo/hail-batch-terra-azure"
```

Comment:
@rtitle Where is this? I'd love to get a look at it.

Reply:
That's the path cached in Leo -- in the Dockerfile this is the pull command:
```
helm pull oci://us-docker.pkg.dev/hail-vdc/terra-dev-public/hail-batch-terra-azure --version 0.1.4 --untar
```